Doc/library/gzip.rst

```diff
@@ -157,13 +157,15 @@ The module defines the following items:
       Accepts a :term:`path-like object`.
 
 
-.. function:: compress(data, compresslevel=9)
+.. function:: compress(data, compresslevel=9, mtime=None)
 
    Compress the *data*, returning a :class:`bytes` object containing
-   the compressed data.  *compresslevel* has the same meaning as in
+   the compressed data.  *compresslevel* and *mtime* has the same meaning as in
    the :class:`GzipFile` constructor above.
 
    .. versionadded:: 3.2
+   .. versionchanged:: 3.8
+      Added *mtime* argument
```

Comment:
Hi @guoci thank you for your update, but in fact, because you add a new parameter, it's a `versionadded` you have to use.

Thank you,

Reply:
I guess this should be `versionchanged` since it introduces a new parameter to the constructor. `versionadded` is used when a new function/method/class itself is added. Reference : https://devguide.python.org/documenting/#paragraph-level-markup

```
 versionadded

    This directive documents the version of Python which added the described
feature, or a part of it, to the library or C API. When this applies to an
entire module, it should be placed at the top of the module section before any
prose.

 versionchanged

    Similar to versionadded, but describes when and what changed in the named
feature in some way (new parameters, changed side effects, platform support,
etc.). This one must have the second argument (explanation of the change).
```

This is used in varied ways through the docs but the above was the intended usage and `versionchanged` should be used for new parameters. There was an issue to correct all these but it was not merged that had a discussion : https://bugs.python.org/issue26366

Some example cases where it was used with different meaning : 

`versionadded` for new argument : 

https://docs.python.org/3/library/datetime.html#datetime.datetime 
New in version 3.6: Added the fold argument.

`versionchanged` for new argument : 

https://docs.python.org/3/library/datetime.html#datetime.datetime.combine
Changed in version 3.6: Added the tzinfo argument.

I am not trying to argue here it's just that I too was little confused over the usage in a related PR :)

Thanks

Reply:
@matrixise is this resolved?